app/controllers/articles.server.controller.js

```diff
@@ -12,8 +12,17 @@ var mongoose = require('mongoose'),
  * Create a article
  */
 exports.create = function(req, res) {
+
+	// Filters out control fields
+	delete req.body.updatedAt;
+	delete req.body.updatedBy;
+	delete req.body.createdAt;
+	delete req.body.createdBy;
+
 	var article = new Article(req.body);
-	article.user = req.user;
+
+	article.createdBy = req.user;
```

Comment:
Just in case some malicious user tries to send that info in the request payload trying to overwrite some private fields in db. If I understand correctly, the new Article(req.body) will try to use any property in body that matches a property in Article, and someone could use this to overwrite fields that are not meant to be overwritten.